README.md

```diff
@@ -125,7 +150,39 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+This was a difficult question and took a lot of time so ill describe the differences here.
+The first one uses the keywords used in the FSM diagram and accounts for 2+ usages of adjectives.
+The second one uses the literal words and DOES NOT account for the chance of 2+ usages of adjectives
+but it produces the 13 entries said in the hint.
+I'm including both out of curiosity more than anything as they both seem valid answers/possibilities to me.
+
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S1        | adjective |     S2
+    S1        |    noun   |     S3
+    S2        | adjective |     S2
+    S2        |    noun   |     S3
+    S3        |    verb   |     S4
+    S4        |    EOI    |     END
+    S4        |   adverb  |     S5
+    S5        |    EOI    |	END
+
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S1        |   smelly  |     S2
+    S1        |   lazy    |     S2
+    S1        |    dog    |     S3
+    S1        |    cat    |     S3
+    S2        |    dog    |     S3
+    S2        |    cat    |     S3
+    S3        |    ran    |     S4
+    S3        |    ate    |     S4
+    S4        |    EOI    |     END
+    S4        |   slowly  |     S5
+    S4        |   noisly  |     S5
+    S5        |    EOI    |	END
```

Comment:
Actually, they're both pretty close. I _did_ want each word spelled out in the table, because that's the way the program is going to want to see it.

The only issue I see with both tables is that you have an unnecessary state: you can merge S1 and S2:

Current State | Next Word | Next State
------------- | --------- | ----------
S0            | "the"     | S1
S1            | "lazy"    | S1
S1            | "smelly"  | S1
S1            | "dog"     | S2
S1            | "cat"     | S2
S2            | "ate      | S3
S2            | "ran"     | S3
S3            | "slowly"  | S4
S3            | "noisily" | S4
S3            | EOI       | END
S4            | EOI       | END

I mentioned in class on Wednesday that the 13 was incorrect, and that my personal answer contained 11 lines.

# -2